buildkitd/entrypoint.sh

```diff
@@ -38,19 +31,51 @@ if [ "$EARTHLY_RESET_TMP_DIR" = "true" ]; then
     rm -rf "${EARTHLY_TMP_DIR:?}"/* || true
 fi
 
-if ! lsmod | grep -wq "^ip_tables"; then
-  echo "Legacy ip_tables is not loaded."
+if [ -z "$IP_TABLES" ]; then
+    echo "Autodetecting iptables"
+
+    if lsmod | grep -wq "^ip_tables"; then
+        echo "Detected iptables-legacy module"
+        IP_TABLES="iptables-legacy"
+
+    elif lsmod | grep -wq "^nf_tables"; then
+        echo "Detected iptables-nft module"
+        IP_TABLES="iptables-nft"
+    else
+        echo "Could not find an ip_tables module; falling back to heuristics."
+
+        legacylines=$(iptables-legacy -t nat -S --wait | wc -l | cut -d' ' -f1)
```

Comment:
is the `cut` really needed here; is there a case where `wc -l` outputs more than one result?
```suggestion
        legacylines=$(iptables-legacy -t nat -S --wait | wc -l)
```

Reply:
`wc -l` returns a filename too. I wanted just the number of lines.

Reply:
See my edit